src/hotspot/share/opto/chaitin.cpp

```diff
@@ -917,6 +918,10 @@ void PhaseChaitin::gather_lrg_masks( bool after_aggressive ) {
             lrg._is_bound = 1;
           }
           break;
+        case Op_RegVMask:
+          lrg.set_num_regs(RegMask::SlotsPerRegVMask);
```

Comment:
This will interfere with non-AVX512 targets, since some of instruction patterns have kReg temporary operands which will still be allocated but not JITed.